six/modules/python/six.sicd/tests/validate_sicd.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+#
+# =========================================================================
+# This file is part of six.sicd-python
+# =========================================================================
+#
+# (C) Copyright 2004 - 2016, MDA Information Systems LLC
+#
+# six.sicd-python is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; If not,
+# see <http://www.gnu.org/licenses/>.
+#
+
+from pysix.six_sicd import *
+from pysix.six_base import *
+from coda.xml_lite import *
+from coda.coda_io import *
+from coda.coda_logging import *
+
+import os
+import sys
+import os.path
+import filecmp
+
+##############################################################
```

Comment:
Update the comments to match what this is really doing.  Probably rename the function to `validateSicd()`.  Would be nice if this script returned 0 or 1 based on if all the SICDs you sent in validated.